update-dependencies.js

```diff
@@ -0,0 +1,211 @@
+'use strict';
+
+require('shelljs/global');
+
+const version = exec('node --version', {silent:true});
+
+console.log('node version: ', version.stdout);
+
+const installBower = exec('npm install -g bower');
+
+const prepBranch = (project, release, lightningThemeRelease) => {
+	return new Promise((resolve, reject) => {
+		const execOpts = {
+			silent: true,
+			async: false,
+			shell: '/bin/bash'
+		};
+
+		const cd = `cd ../${project}`
+		const gitstuff = `git stash && git checkout master && git pull upstream master && git status`
+
+		const command = String.raw`${cd} && ${gitstuff}`;
+
+		// npm & bower update
+		const runUpdates = exec(command,
+			execOpts,
+			(code, stdout, stderr) => {
+				// check to see if status is clean
+				// console.log(stdout.match('nothing to commit, working directory clean'));// grep instead
+				// console.log(stdout);
+				if (code === 0 && stdout.match('nothing to commit, working directory clean').length > 0) {
+					// console.log(project, 'updateDependencyRequirements successful');
+					resolve(project, release, lightningThemeRelease);
+				} else {
+					console.log('Exit code:', code);
+					console.log(project, 'had errors when updateDependencyRequirements.')
+					console.log('Program output:', stdout);
+					console.log('Program stderr:', stderr);
+					reject(project, release, lightningThemeRelease);
+				}
+			}
+		);
+	});
+}
+
+const updateDependencyRequirements = (project, release, lightningThemeRelease) => {
+	// console.log('updateDependencyRequirements', project);
+	return new Promise((resolve, reject) => {
+		const execOpts = {
+			silent: true,
+			async: false,
+			shell: '/bin/bash'
+		};
+
+		const cd = `cd ../${project}`
+
+		const packageFUX = String.raw`sed -i '' 's/\(fuelux": "\)[\^\>\=\~\<0-9.x]*\("\)/fuelux": "${release}"/g' ./package.json`;
+		const bowerFUX = String.raw`sed -i '' 's/\(fuelux": "\)[\^\>\=\~\<0-9.x]*\("\)/fuelux": "${release}"/g' ./bower.json`;
+
+		const packageFLTPrefix = String.raw`"fuelux-lightning-theme": "git+ssh:\/\/git@github.exacttarget.com:uxarchitecture\/fuelux-lightning-theme.git\#`;
```

Comment:
Lots of internal references. @futuremint is this a problem? should this tooling go somewhere else?